packages/components/organisms/f-registration/src/components/Registration.vue

```diff
@@ -428,8 +473,26 @@ export default {
         },
 
         isFormInvalid () {
-            this.$v.$touch();
-            return this.$v.$invalid;
+            const v = this.$v;
```

Comment:
Could you add a JSDoc comment to this, as the function has changed a fair bit from its initial creation.